src/main/resources/avro/wip/variationReference.avdl

```diff
@@ -0,0 +1,263 @@
+@namespace("org.ga4gh.wip")
```

Comment:
Hmm so should I remove it?

Looking at the graph branch, this file exists.
https://github.com/ga4gh/schemas/blob/graph/src/main/resources/avro/wip/variationReference.avdl